code/modules/mob/living/brain/posibrain.dm

```diff
@@ -41,15 +41,26 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 			GLOB.posibrain_notify_cooldown = world.time + askDelay
 
 /obj/item/device/mmi/posibrain/attack_self(mob/user)
-	if(!brainmob || brainmob.key)
+	if(!brainmob)
+		to_chat(user, "<span class='warning'>[src]'s personality matrix is corrupted! Attempting to reboot...</span>")
+		brainmob = new(src)
+		if(!brainmob) //something went really wrong!
```

Comment:
This is an insanity check; it will never happen, and accordingly should be removed.